<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShortcutDialog</class>
 <widget class="QWidget" name="ShortcutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>523</width>
       <height>335</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widget{
    background-image: url(&quot;:/images/widgetbackground.png&quot;);
}

QLineEdit{
	border-radius: 4px;
	border: 2px solid #00fcfc;
	background-color: rgba(0.0, 98.7, 98.9, 0.5);
	color: #00fcfc;
}

/*QLineEdit:focus{
	background-color: #00fcfc;
	color: #012a4a;
}*/

QPushButton{
border-radius: 4px;
	border: 2px solid #00fcfc;
	background-color: rgba(0.0, 98.7, 98.9, 0.5);
	color: #00fcfc;
}
QPushButton#saveButton, QPushButton#cancelButton{
width: 70px;
}
QPushButton:hover{
	background-color: #00fcfc;
color: #012a4a;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>38</number>
        </property>
        <property name="topMargin">
         <number>24</number>
        </property>
        <property name="rightMargin">
         <number>28</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="shortcutNameEdit">
          <property name="placeholderText">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="argList">
          <property name="placeholderText">
           <string>List of argument</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="executablePathEdit">
            <property name="placeholderText">
             <string>File path</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="executablePathButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="iconPathEdit">
            <property name="placeholderText">
             <string>Icon path</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="chooseIconButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="cancelButton">
            <property name="text">
             <string>Cancel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveButton">
            <property name="text">
             <string>Ok</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
